tests/web/test_transactions.py

```diff
@@ -270,10 +265,12 @@ def test_post_invalid_transaction(mock_logger, client, exc, msg, monkeypatch,):
     def mock_validation(self_, tx):
         raise exc_cls(msg)
 
+    TransactionMock = Mock(validate=mock_validation)
+
     monkeypatch.setattr(
         'bigchaindb.Bigchain.validate_transaction', mock_validation)
     monkeypatch.setattr(
-        'bigchaindb.models.Transaction.from_dict', lambda tx: None)
+        'bigchaindb.models.Transaction.from_dict', lambda tx: TransactionMock)
```

Comment:
Could you please elaborate this?

Reply:
The test mocks the validation that happens inside `Transaction.validate` now. I can actually also remove the previous patch.